### DISCUSSION TITLE
Mushroom Crates

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#686] [Ideas]
It would be cool if, like the other crops, mushrooms could also be compacted into crates.

**Comment by vectorwing** [comment]
I appreciate your enthusiasm with ideas, but perhaps try merging similar ones in the same posts. You created quite a few threads at once... 😅

Crates are an idea I borrowed from Quark originally; I restricted the variety of crates FD adds because I didn't want to steal their concept completely. Given they don't add egg or mushroom crates, this could happen one day; I just didn't judge mushrooms that important to hoard at the time.

**Reply by Aceplante** [reply]
Yea but they are used in a lot of recipes specifically dumplings and can be used for cabbage rolls as well. Plus you can farm them just like any other crop using rich soil. So I have a lot and a crate would be helpful
